_notebooks/2021-02-09-task-spooler.ipynb

```diff
@@ -0,0 +1,2746 @@
+{
```

Comment:
<p>I would remove this output otherwise the rendering would be extremely long. Use <code>%%capture</code>  or any or shell command to make it short</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LibreCV/blog/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>i guess using <code>#collapse-hide</code> here is fine? </p>

Reply:
<p>or <code>#collapse-output</code>?</p>

Reply:
We are experimenting here so I guess anything that generates a good looking and informative output should be fine to be keep it.